packages/superset-ui-preset-chart-xy/types/@vx/legend/index.d.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/no-multi-comp */
+declare module '@vx/legend' {
+  import React from 'react';
+
+  export function LegendOrdinal(props: { [key: string]: any }): React.ReactElement;
```

Comment:
just noting a mix of `ReactElement`/`ReactNode`, I'm not sure which is best TBH. May just have to experiment and see what happens 🔬

Reply:
> React.Node. This represents any node that can be rendered in a React application. React.Node can be null, a boolean, a number, a string, a React element, or an array of any of those types recursively. If you need a return type for your component render() methods then you should use React.Node .

Hmm, maybe I should change these to `ReactNode`